packages/typespec-test/test/openai_generic/generated/typespec-ts/review/openai-generic.api.md

```diff
@@ -534,6 +655,37 @@ export interface OpenAIFile {
+// src/classic/files/index.ts:31:5 - (ae-forgotten-export) The symbol "RetrieveOptions_2" needs to be exported by the entry point index.d.ts
+// src/classic/files/index.ts:35:5 - (ae-forgotten-export) The symbol "DeleteOptions" needs to be exported by the entry point index.d.ts
+// src/classic/files/index.ts:39:5 - (ae-forgotten-export) The symbol "DownloadOptions" needs to be exported by the entry point index.d.ts
+// src/classic/fineTunes/index.ts:26:5 - (ae-forgotten-export) The symbol "CreateOptions_9" needs to be exported by the entry point index.d.ts
+// src/classic/fineTunes/index.ts:30:5 - (ae-forgotten-export) The symbol "ListOptions_3" needs to be exported by the entry point index.d.ts
+// src/classic/fineTunes/index.ts:31:5 - (ae-forgotten-export) The symbol "RetrieveOptions_3" needs to be exported by the entry point index.d.ts
+// src/classic/fineTunes/index.ts:35:5 - (ae-forgotten-export) The symbol "ListEventsOptions_2" needs to be exported by the entry point index.d.ts
+// src/classic/fineTunes/index.ts:39:5 - (ae-forgotten-export) The symbol "CancelOptions_2" needs to be exported by the entry point index.d.ts
+// src/classic/fineTuning/jobs/index.ts:26:5 - (ae-forgotten-export) The symbol "CreateOptions_4" needs to be exported by the entry point index.d.ts
+// src/classic/fineTuning/jobs/index.ts:30:5 - (ae-forgotten-export) The symbol "ListOptions" needs to be exported by the entry point index.d.ts
+// src/classic/fineTuning/jobs/index.ts:33:5 - (ae-forgotten-export) The symbol "RetrieveOptions" needs to be exported by the entry point index.d.ts
+// src/classic/fineTuning/jobs/index.ts:37:5 - (ae-forgotten-export) The symbol "ListEventsOptions" needs to be exported by the entry point index.d.ts
+// src/classic/fineTuning/jobs/index.ts:41:5 - (ae-forgotten-export) The symbol "CancelOptions" needs to be exported by the entry point index.d.ts
+// src/classic/images/index.ts:22:5 - (ae-forgotten-export) The symbol "CreateOptions_10" needs to be exported by the entry point index.d.ts
+// src/classic/images/index.ts:26:5 - (ae-forgotten-export) The symbol "CreateEditOptions" needs to be exported by the entry point index.d.ts
+// src/classic/images/index.ts:30:5 - (ae-forgotten-export) The symbol "CreateVariationOptions" needs to be exported by the entry point index.d.ts
+// src/classic/models/index.ts:21:5 - (ae-forgotten-export) The symbol "ListOptions_4" needs to be exported by the entry point index.d.ts
+// src/classic/models/index.ts:22:5 - (ae-forgotten-export) The symbol "RetrieveOptions_4" needs to be exported by the entry point index.d.ts
+// src/classic/models/index.ts:23:5 - (ae-forgotten-export) The symbol "DeleteOptions_2" needs to be exported by the entry point index.d.ts
+// src/classic/moderations/index.ts:13:5 - (ae-forgotten-export) The symbol "CreateOptions_11" needs to be exported by the entry point index.d.ts
```

Comment:
Are we going to do something about these warnings ? If yes, we will have to resolve the operation options conflict.